app/components/task-details.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="task-details">
+  <div class="task-title"><b>Title: </b>{{@task.title}}</div>
+  <div class="task-purpose"><b>Purpose: </b>{{@task.purpose}}</div>
```

Comment:
Can we follow BEM conventions?

Reply:
Will have to read up about it. 

I didn't change the CSS naming which was done by @sumitd94 earlier.

Reply:
Sure, it's not a big thing but in some files I see BEM being followed and in some I don't

Reply:
done